src/Quaternion.jl

```diff
@@ -118,7 +118,7 @@ function exp(q::Quaternion)
     if th > 0
         scale *= sin(th) / th
     end
-    Quaternion(se * cos(th), scale * q.v1, scale * q.v2, scale * q.v3, abs(s) < eps(typeof(s)))
+    Quaternion(se * cos(th), scale * q.v1, scale * q.v2, scale * q.v3, iszero(abs(s)))
```

Comment:
No need for `abs` here:
```suggestion
    Quaternion(se * cos(th), scale * q.v1, scale * q.v2, scale * q.v3, iszero(s))
```

Reply:
Oh, that's my mistake. Thanks for the suggestion!